src/demo-app/expansion/expansion-demo.html

```diff
@@ -0,0 +1,58 @@
+<h1>Single Expansion Panel</h1>
+<md-expansion-panel class="md-expansion-demo-width" #myPanel>
+  <md-expansion-panel-header>
+    <span class="description">This is a panel description.</span>
+    <span class="title">Panel Title</span>
```

Comment:
If title and description are part of the template API they should be elements like `md-panel-header-title`, `md-panel-header-description`
(sorry, I had previously thought you were just styling something in the expansion panel's template)